megamek/src/megamek/common/actions/WeaponAttackAction.java

```diff
@@ -4274,8 +4277,7 @@ else if ((atype != null)
             ToHitData immobileMod = Compute.getImmobileMod(target, aimingAt, aimingMode);
             // grounded dropships are treated as immobile as well for purpose of
             // the mods
-            if ((null != te) && !te.isAirborne() && !te.isSpaceborne() && (te instanceof Dropship)
-                    && ((Aero) te).isSpheroid()) {
+            if ((null != te) && !te.isAirborne() && !te.isSpaceborne() && (te instanceof Dropship)) {
                 immobileMod = new ToHitData(-4, Messages.getString("WeaponAttackAction.ImmobileDs"));
             }
```

Comment:
This code is odd to me... why are we just overriding when it could instead be an else/if statement that assigns a value to the immobileMod. That would decrease the processing required